bluesky_queueserver/manager/plan_queue_ops.py

```diff
@@ -915,8 +914,8 @@ async def set_processed_plan_as_stopped(self, exit_status):
         Returns
         -------
         dict
-            The plan added to the history including ``exit_status``. If another no plan is currently
-            running, then ``{}`` is returned.
+            The plan added to the history including ``exit_status``. If another item (plan)
```

Comment:
Should the `plan` be replaced with `item` here?

Reply:
At this point the plan history contains only plans.

Reply:
I don't think it is critical, because currently only plans may have 'running' status. Execution of the one available instruction is instant. But let's generalize it.